.github/workflows/ci.yaml

```diff
@@ -303,29 +303,9 @@ jobs:
       
       - name: Create AVD and generate snapshot for caching
         if: steps.avd-cache.outputs.cache-hit != 'true'
-        uses: reactivecircus/android-emulator-runner@v2
-        with:
-          api-level: ${{ matrix.api-level }}
-          arch: ${{ matrix.arch }}
-          force-avd-creation: false
-          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          disable-animations: false
-          script: echo "Generated AVD snapshot for caching."
-      
-      - run: yarn install
-      
-      - run: nohup yarn react-native start > "yarn_react_native_start.out" &
-      
-      - name: Run Detox Build
-        run: yarn detox build -c android.emu.debug -n test
-      
-      - name: Run Detox Test
-        uses: reactivecircus/android-emulator-runner@v2
-        with:
-          api-level: ${{ matrix.api-level }}
-          arch: ${{ matrix.arch }}
-          force-avd-creation: false
-          emulator-options: -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          disable-animations: true
-          script: yarn detox test -c android.emu.debug -n test
-      
+        run: ./scripts/e2e_tests.sh -a ${{ matrix.avd-arch }} -s
```

Comment:
matrix needs to be changed to match integration tests (with `avd-arch` and `abi`  here:
https://github.com/zingolabs/zingo-mobile/pull/330/files/5eb69bf6e343c99e65869837ff3b8d6ba816505b..9490be55d7cd99c4f49e1a7685bec8e22be4617c#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR266